Libplanet/Net/Swarm.cs

```diff
@@ -1129,6 +1129,14 @@ private void ProcessMessageHandler(object target, Message message)
                         break;
                     }
 
+                case BlockHeaderMessage blockHeader:
+                    {
+                        Task.Run(
+                            async () => await ProcessBlockHeader(blockHeader, _cancellationToken),
```

Comment:
Does it differ from `Task.Run(ProcessBlockHeader(blockHeader, _cancellationToken), _cancellationToken)`?

Reply:
It throws exception because `Task.Run()` takes an action as its argument but `ProcessBlockHeader(blockHeader, _cancellationToken)` is `Task`.